apps/api/src/app/invites/usecases/resend-invite/resend-invite.usecase.ts

```diff
@@ -0,0 +1,58 @@
+import { Injectable, Scope } from '@nestjs/common';
+import { OrganizationRepository, UserRepository, MemberRepository } from '@novu/dal';
+import { MemberStatusEnum } from '@novu/shared';
+import { Novu } from '@novu/node';
+import { ApiException } from '../../../shared/exceptions/api.exception';
+import { ResendInviteCommand } from './resend-invite.command';
+import { capitalize, createGuid } from '../../../shared/services/helper/helper.service';
+
+@Injectable({
+  scope: Scope.REQUEST,
+})
+export class ResendInvite {
+  constructor(
+    private organizationRepository: OrganizationRepository,
+    private userRepository: UserRepository,
+    private memberRepository: MemberRepository
+  ) {}
+
+  async execute(command: ResendInviteCommand) {
+    const organization = await this.organizationRepository.findById(command.organizationId);
+    if (!organization) throw new ApiException('No organization found');
+
+    const foundInvitee = await this.memberRepository.findById(command.memberId);
```

Comment:
```suggestion
    const foundInvitee = await this.memberRepository.findOne({ _id: command.memberId, _organizationId: command.organizationId });
```

Could we add a key from the token here just to make sure people are not passing memberIds related to other organization since we operate in a multi-tenant environment.

Reply:
Yes, great finding!